nbdime/diffing/notebooks.py

```diff
@@ -194,6 +195,8 @@ def diff_mime_bundle(a, b, path=None):
     "/cells/*": diff,
     "/cells/*/outputs": diff_sequence_multilevel,
     "/cells/*/outputs/*": diff_single_outputs,
+    "/cells/*/attachments": diff_sequence_multilevel,
+    "/cells/*/attachments/*": diff_mime_bundle,
```

Comment:
I'm inclined to skip that at least for now, we should focus on polishing and making existing features robust now. Agree that a rename op would be much easier than an array move op though, but it still needs to be handled across quite a few places in the codebase.